ICS-ICAL Sync/Code.gs

```diff
@@ -4,20 +4,34 @@
 * 2) Changes lines 11-20 to be the settings that you want to use
 * 3) Click in the menu "Run" > "Run function" > "Install" and authorize the program
 *    (For steps to follow in authorization, see this video: https://youtu.be/_5k10maGtek?t=1m22s )
-*
+* 4) To stop Script from running click in the menu "Edit" > "Current Project's Triggers".  Delete the running trigger.
 */
 
 // --------------- SETTINGS ---------------
-var sourceCalendarURL = ""; //The ics/ical url that you want to get events from
-var targetCalendarName = ""; //The name of the Google Calendar you want to add events to
-var howFrequent = 15; //What interval (minutes) to run this script on to check for new events
+
+// var sourceCalendars = { targetCalendarName:"sourceCalendarURL"}
+//     "targetCalendarName" is The name of the Google Calendar you want to add events to
+//     "sourceCalendarURL" is The ics/ical url that you want to get events from
+
+var sourceCalendars={
```

Comment:
Let's do this change (support for multiple calendars) in a separate pull request. Log an issue for it